app/controllers/solutions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SolutionsController < ApplicationController
+
+  def create
+    @solution = Solution.new(solution_params)
+    @solutions = @question.solutions
```

Comment:
まず，`@question` を定義して，その質問の回答として`@solution`を決めなければなりません。

```suggestion
    @question = Question.find(params[:question_id])
    @solution = @question.solutions.build(solution_params)
```